### PR TITLE
Fix typo in thumbnail

### PR DESCRIPTION
**Pull request by tarikcoskun**
tail**d**windcss -> tailwindcss

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-![Elecrue Banner](https://user-images.githubusercontent.com/63385587/133573100-6c8f5ad2-028d-4a21-a461-ae81c277c20c.png)
+![Elecrue Banner](https://user-images.githubusercontent.com/44239968/133919975-a0fe079f-4057-4cb6-bd48-80668fa59bfa.png)
```
Comment (barbarbar338) [inline README.md:1]: Approved 👍🏻

Comment (RajvirSingh1313) [reply README.md:1]: Thanks, I make alot of slily mistakes, Thanks alot

Comment (tarikcoskun) [reply README.md:1]: Not a problem, good luck with your project!!

Comment (RajvirSingh1313) [reply README.md:1]: Thanks a lot, thanks for contributing looking forward to work with you!

**Pull request merged**